types/node/console.d.ts

```diff
@@ -1,10 +1,26 @@
 declare module 'console' {
     import { InspectOptions } from 'util';
 
+    namespace console {
```

Comment:
There are a lot of declaration gymnastics going on here. It might be clearer if the `namespace console` were moved into the `global {}` declaration, and the export just be `export = globalThis.console`.

Reply:
One of the goals here was to remove global declarations but I guess this is a reasonable tradeoff.